lib/markdownlint-provider.coffee

```diff
@@ -15,10 +18,19 @@ module.exports =
 
       bufferText = textEditor.getText()
 
+      filePath = textEditor.getPath()
+
       options = {
         "strings": { "string": bufferText }
       }
 
+      configPath = find filePath, '.markdownlintrc'
+
+      if configPath
+        configText = fs.readFileSync(configPath, 'utf8')
+        options['config'] = rc.parse(configText)
```

Comment:
it can be [json](https://www.npmjs.com/package/rc#formatted-as-json) or [INI](https://www.npmjs.com/package/rc#formatted-as-ini).

I don't like adding unnecessary dependencies either, but by adding the exact same parsing library that [markdownlint-cli](https://github.com/igorshubovych/markdownlint-cli) uses, it ensures the same expected behavior.